src/sentry/static/sentry/app/components/charts/eventsChart.tsx

```diff
@@ -88,6 +106,22 @@ class Chart extends React.Component<ChartProps> {
     return AreaChart;
   }
 
+  handleLegendSelectChanged = legendChange => {
+    const {selected} = legendChange;
+    const seriesSelection = Object.keys(selected).reduce((state, key) => {
+      // we only want them to be able to disable the Releases series,
+      // and not any of the other possible series here
+      state[key] = key === 'Releases' ? selected[key] : true;
```

Comment:
One caveat of this is that in the legend, the other series will appear clickable but nothing will happen.

Reply:
How does that look in top5 mode when the series names are long? Does echarts truncate the names?

Reply:
The series name in top 5 mode are still not show which is also the current behavior. See screenshot

![Screen Shot 2020-07-14 at 10 16 11 AM](https://user-images.githubusercontent.com/10239353/87436459-10d25f00-c5bb-11ea-910c-f4838aa687bf.png)